### PR TITLE
Remove signalr app

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <HtmlJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.html.json"</HtmlJobs>
     <JsonJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.json.json"</JsonJobs>
     <MultiQueryJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.multiquery.json"</MultiQueryJobs>
-    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json -t SignalR</SignalRJobs>
+    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json -t SignalR -r signalr --projectFile benchmarks/BenchmarkServer/BenchmarkServer.csproj</SignalRJobs>
```
Comment (sebastienros) [inline build/repo.targets]: I assume this is the name of your web app in the signalr repository?

Comment (BrennanConroy) [reply build/repo.targets]: That's the current name in https://github.com/aspnet/SignalR/pull/1579

**Comment by sebastienros** [comment]
Ok to merge when the SignalR repos is updated

File: src/Benchmarks/signalr.json [hunk 2]
```diff
@@ -11,6 +11,6 @@
     "Warmup": 2
   },
   "SignalRBroadcast": {
-    "Path": "/signalr/broadcast/default"
```
Comment (davidfowl) [inline src/Benchmarks/signalr.json]: I think we should move this to the SignalR repo

File: build/repo.targets [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <HtmlJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.html.json"</HtmlJobs>
     <JsonJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.json.json"</JsonJobs>
     <MultiQueryJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.multiquery.json"</MultiQueryJobs>
-    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json -t SignalR -r signalr --projectFile benchmarks/BenchmarkServer/BenchmarkServer.csproj</SignalRJobs>
+    <SignalRJobs>-j https://raw.githubusercontent.com/aspnet/SignalR/brecon/benchmark/benchmarks/BenchmarkServer/signalr.json -t SignalR -r signalr@brecon/benchmark --projectFile benchmarks/BenchmarkServer/BenchmarkServer.csproj</SignalRJobs>
```
Comment (sebastienros) [inline build/repo.targets]: `brecon`, is that just for testing? You should merge your PR before merging this one.

Comment (BrennanConroy) [reply build/repo.targets]: Yeah, I'll change these values when the signalr change goes in

**Pull request merged**